app/models/operatingsystem.rb

```diff
@@ -60,6 +60,7 @@ class Operatingsystem < ApplicationRecord
 
   FAMILIES = { 'Debian'    => %r{Debian|Ubuntu}i,
                'Redhat'    => %r{RedHat|Centos|Fedora|Scientific|SLC|OracleLinux}i,
+               'Redhat'    => %r{RedHat|Centos|Fedora|Scientific|SLC|OracleLinux|Rocky}i,
```

Comment:
Looks like a bad CP?

Reply:
Well depends, it's correct CP, as the alma linux is in separate PR, I can join 'em though. It would probably make sense